ocs-ktc/src/test/kotlin/org/ostelco/prime/ocs/parser/UserLocationTest.kt

```diff
@@ -30,6 +30,18 @@ class UserLocationTest {
         assertEquals("16", userLocation?.mnc)
     }
 
+    @Test
+    fun userLocationParserMalaysiaSai() {
+
+        val locationHexString = "0105F26182BF804E"
```

Comment:
What does his magic hex string mean?

Reply:
This is the data value we get in the 3GPP-USER-LOCATION-INFO AVP.